PocketKit/Sources/PocketKit/Home/HomeViewModel.swift

```diff
@@ -287,6 +288,16 @@ extension HomeViewModel {
     }
 }
 
+// MARK: - Loading Section
+extension HomeViewModel {
+    static func buildLoadingSnapshot() -> Snapshot {
```

Comment:
I believe it's because we were originally calling it from the initializer before all the other properties on `self` were initialized, so we couldn't pull it out to an instance method (since that would reference self before being fully initialized)